autopkg/autopkg_tools.py

```diff
@@ -72,35 +97,40 @@ def _parse_report(self, report):
         return {"imported": imported_items, "failed": failed_items}
 
     def run(self):
-        report = "/tmp/autopkg.plist"
-        if not os.path.isfile(report):
-            # Letting autopkg create them has led to errors on github runners
-            Path(report).touch()
-
-        try:
-            cmd = [
-                "/usr/local/bin/autopkg",
-                "run",
-                self.path,
-                "-v",
-                "--post",
-                "io.github.hjuutilainen.VirusTotalAnalyzer/VirusTotalAnalyzer",
-                "--report-plist",
-                report,
-            ]
-            cmd = " ".join(cmd)
-            if DEBUG:
-                print("Running " + str(cmd))
+        if self.verified == False:
```

Comment:
We allow for `self.verify()` never having been run since this block is no-op on `self.verified == None`.